_maps/shuttles/emergency/emergency_theatre.dmm

```diff
@@ -0,0 +1,970 @@
+	pixel_x = 1
+	},
+/obj/item/clothing/mask/cigarette/cigar/cohiba{
+	pixel_x = 8
+	},
+/obj/item/clothing/mask/cigarette/cigar/cohiba{
+	pixel_x = 8
+	},
+/obj/item/clothing/mask/cigarette/cigar/cohiba{
+	pixel_x = 8
+	},
+/turf/open/floor/carpet/black,
+/area/shuttle/escape)
+"JG" = (
+/obj/machinery/light/floor{
+	brightness = 15;
+	bulb_colour = "#FFE4CE";
+	bulb_emergency_brightness_mul = 0.25;
+	bulb_vacuum_brightness = 15;
+	nightshift_brightness = 10;
+	nightshift_light_color = "#E6EBFF"
+	},
+/turf/open/floor/wood,
+/area/shuttle/escape)
+"JO" = (
+/obj/structure/table/wood/fancy/royalblue,
+/obj/item/reagent_containers/food/drinks/drinkingglass{
+	pixel_y = 9;
+	pixel_x = -6
+	},
+/obj/item/reagent_containers/food/drinks/drinkingglass{
+	pixel_y = 9;
+	pixel_x = 6
+	},
+/obj/item/reagent_containers/food/drinks/bottle/champagne{
+	pixel_y = 11
+	},
+/obj/item/reagent_containers/food/drinks/drinkingglass{
+	pixel_y = 4;
+	pixel_x = 6
+	},
+/obj/item/reagent_containers/food/drinks/drinkingglass{
+	pixel_y = 4;
+	pixel_x = -6
+	},
+/turf/open/floor/carpet/royalblue,
+/area/shuttle/escape)
+"LQ" = (
+/obj/structure/statue/sandstone/venus,
+/turf/open/floor/carpet/black,
+/area/shuttle/escape)
+"Nn" = (
+/obj/structure/table/wood/fancy/black,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/obj/item/clothing/glasses/monocle,
+/turf/open/floor/carpet/black,
+/area/shuttle/escape)
+"NH" = (
+/obj/machinery/light{
+	dir = 4
+	},
+/turf/open/floor/carpet/black,
+/area/shuttle/escape)
+"OI" = (
+/obj/item/storage/fancy/cigarettes/cigars/havana,
+/obj/structure/table/wood/fancy/royalblue,
+/obj/item/storage/box/matches{
+	pixel_x = -10;
+	pixel_y = 7
+	},
+/turf/open/floor/carpet/royalblue,
+/area/shuttle/escape)
+"Pk" = (
+/obj/machinery/door/airlock/medical/glass{
+	name = "Escape Shuttle Infirmary"
+	},
+/turf/open/floor/carpet/cyan,
+/area/shuttle/escape)
+"PH" = (
+/turf/open/floor/carpet/cyan,
+/area/shuttle/escape)
+"Rv" = (
+/obj/structure/shuttle/engine/propulsion,
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/escape)
+"RK" = (
+/obj/structure/chair/fancy/corp{
+	dir = 4
+	},
+/turf/open/floor/carpet/black,
+/area/shuttle/escape)
+"Sk" = (
+/obj/structure/extinguisher_cabinet{
+	pixel_y = 29
+	},
+/obj/machinery/light{
+	dir = 1
+	},
+/turf/open/floor/carpet/royalblue,
+/area/shuttle/escape)
+"SP" = (
+/turf/open/floor/mineral/plastitanium/red/brig,
+/area/shuttle/escape)
+"Tx" = (
+/obj/structure/chair/fancy/corp{
```

Comment:
Can everyone see the stage from the back? All the seating is kinda too much if it extends to where people cant see the stage.

Reply:
![imagen](https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/43dcf036-ad53-471d-a97e-375efa113905)

you are right, they couldnt, I made it smaller